src/adapters/http/client.ts

```diff
@@ -0,0 +1,52 @@
+import HttpEventEmitter from "../../lib/httpEventEmitter";
+import GleeMessage from "../../lib/message.js";
+import Adapter from "../../lib/adapter.js";
+
+class HttpClientAdapter extends Adapter {
+  async connect(): Promise<this> {
+    return this._connect();
+  }
+  _connect(): Promise<this> {
+    return new Promise(async (resolve, reject) => {
+      const requestObj = {
+        method: "POST",
+        url: "http://localhost:8081/trendingAnime",
+        body: {
+          name: "trendingAnime",
+          rating: 5,
+          studio: "teststudio",
+          genre: "testgenre",
+        },
+        query:{
+          name: "trendingAnime",
+          rating: "5",
+          studio: "teststudio",
+          genre: "testgenre",
+        }
+      };
+      this.emit("send", requestObj);
+    });
```

Comment:
Is it being called from functions, I can see that it is being called when the client first starts but does it work when you return a `send` operation from the functions?